src/main/scala/org/broadinstitute/dsde/snoop/Boot.scala

```diff
@@ -13,12 +13,21 @@ import com.typesafe.config.ConfigFactory
 import java.io.File
 import scala.reflect.runtime.universe._
 
+import scala.sys.process.Process
+
 object Boot extends App {
+  val conf = ConfigFactory.parseFile(new File("/etc/snoop.conf"))
+  //set system properties for ssl db connection
```

Comment:
this new block of code should be in its own method named according to this comment